packages/ember-views/lib/views/text_field.js

```diff
@@ -36,33 +35,21 @@ function canSetTypeOfInput(type) {
 }
 
 function getTypeComputed() {
-  if (Ember.FEATURES.isEnabled('new-computed-syntax')) {
-    return computed({
-      get() {
-        return 'text';
-      },
+  return computed({
```

Comment:
Would you mind moving this down to where it is used inline (sorry for the troubles)?  I only moved it out into a function so that it could return the correctly formatted `computed` instance.